webapp/src/translations/languages/en.json

```diff
@@ -1015,7 +1015,7 @@
   "general": {
     "defiApp": "DeFi Wallet",
     "updateFailed": "An error occurred while checking for a software update. Please manually download the latest release from defichain.com. (Only download the app from official sources)",
-    "quitingApp": "Quitting DefiChain app...",
+    "quitingApp": "Quitting DefiChain app ...",
```

Comment:
"Quitting DeFiChain app..." should remain as "Quitting DeFiChain app...", no space. However, "DefiChain" should be spelt "DeFiChain", appreciate if you could help with this adjustment. Thank you!

Reply:
You are right. I did some research and removed the space before the ellipsis. I also changed the spelling to DeFiChain.